Sources/CodeScanner/CodeScanner.swift

```diff
@@ -163,11 +164,14 @@ public struct CodeScannerView: UIViewControllerRepresentable {
             view.backgroundColor = UIColor.black
             captureSession = AVCaptureSession()
 
-            guard let videoCaptureDevice = AVCaptureDevice.default(for: .video) else { return }
+            if videoCaptureDevice == nil{
```

Comment:
This could still be a guard statement

```suggestion
            guard let videoCaptureDevice = videoCaptureDevice else {
```
```suggestion
            if videoCaptureDevice == nil{
```